apps/src/assetManagement/AssetManager.jsx

```diff
@@ -135,7 +135,7 @@ module.exports = React.createClass({
 
       assetList = (
         <div>
-          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0', paddingRight: '15px'}}>
+          <div style={{maxHeight: '330px', overflowY: 'scroll', margin: '1em 0'}}>
```

Comment:
Also, these styles could be extracted into a styles object like elsewhere (assuming we end up agreeing that this is the preferred approach).